Index: src/drivers/net/ethernet/mellanox/mlx5/core/en/reporter_rx.c
===================================================================
--- src.orig/drivers/net/ethernet/mellanox/mlx5/core/en/reporter_rx.c	2020-07-14 20:14:17.355462304 +0200
+++ src/drivers/net/ethernet/mellanox/mlx5/core/en/reporter_rx.c	2020-07-14 20:14:58.124525484 +0200
@@ -222,8 +222,7 @@
 }
 
 static int mlx5e_rx_reporter_recover(struct devlink_health_reporter *reporter,
-				     void *context,
-				     struct netlink_ext_ack *extack)
+				     void *context)
 {
 	struct mlx5e_priv *priv = devlink_health_reporter_priv(reporter);
 	struct mlx5e_err_ctx *err_ctx = context;
@@ -302,8 +301,7 @@
 }
 
 static int mlx5e_rx_reporter_diagnose(struct devlink_health_reporter *reporter,
-				      struct devlink_fmsg *fmsg,
-				      struct netlink_ext_ack *extack)
+				      struct devlink_fmsg *fmsg)
 {
 	struct mlx5e_priv *priv = devlink_health_reporter_priv(reporter);
 	struct mlx5e_params *params = &priv->channels.params;
