Index: src/drivers/net/ethernet/mellanox/mlx5/core/backport_compat.h
===================================================================
--- src.orig/drivers/net/ethernet/mellanox/mlx5/core/backport_compat.h	2020-07-14 20:20:22.753886784 +0200
+++ src/drivers/net/ethernet/mellanox/mlx5/core/backport_compat.h	2020-07-14 20:23:18.712075566 +0200
@@ -23,4 +23,7 @@
 
 #define DEVLINK_PORT_FLAVOUR_VIRTUAL DEVLINK_PORT_FLAVOUR_PHYSICAL
 
+
+#define TC_SETUP_FT 14
+
 #endif
Index: src/drivers/net/ethernet/mellanox/mlx5/core/en_rep.c
===================================================================
--- src.orig/drivers/net/ethernet/mellanox/mlx5/core/en_rep.c	2020-07-14 20:17:21.652692483 +0200
+++ src/drivers/net/ethernet/mellanox/mlx5/core/en_rep.c	2020-07-14 20:24:05.088125322 +0200
@@ -50,6 +50,7 @@
 #include "lib/port_tun.h"
 #define CREATE_TRACE_POINTS
 #include "diag/en_rep_tracepoint.h"
+#include "backport_compat.h"
 
 #define MLX5E_REP_PARAMS_DEF_LOG_SQ_SIZE \
 	max(0x7, MLX5E_PARAMS_MINIMUM_LOG_SQ_SIZE)
